browser/src/Renderer/CanvasRenderer.ts

```diff
@@ -278,12 +279,16 @@ export class CanvasRenderer implements INeovimRenderer {
         const delta = boundsStartX - normalizedBoundsStartX
         const normalizedBoundsWidth = Math.ceil(boundsWidth + delta)
 
+        const normalizedBoundsY = Math.floor(boundsY)
```

Comment:
The height fix seems reasonable to me, though - we add a pixel padding in between the lines anyway, so some fraction to round it up to a whole number is good. Thanks for looking at this!